build.sbt

```diff
@@ -64,3 +64,16 @@ lazy val hmrcMongoTest = Project("hmrc-mongo-test", file("hmrc-mongo-test"))
       Resolver.typesafeRepo("releases")
     )
   )
+
+lazy val metrixPlay26 = Project("metrix", file("metrix"))
```

Comment:
does it make sense to add metricsPlay27 as well?

Reply:
It does, I'll push an update to create an `hmrc-mongo-metrix-common` and the two separate play26/7 variants